datepicker black fixed

esdebon opened the pull request:
Line 197 change blackColor to whiteColor, with this the datepicker in iOS7is fixed

esdebon committed:
datepicker black fixed

Line 197 change blackColor to whiteColor, with this the datepicker black is fixed

Commit:
Compiled ANE with Datepicker black fixed

Compiled ANE with Datepicker black fixed

esdebon committed:
DatePicker works in landscape mode for Android

DatePicker works in landscape mode for Android change merge from Benjamin Durin

mola2alex commented:
Where can I get the updated ANE?

esdebon commented:
Here:

https://github.com/esdebon/NativeDialogs

mola2alex commented:
Thanks, I found it and it seems to work.  Awesome job.  Now if only the native alert could handle more than 2 buttons like the old one...

mola2alex commented:
It seemed to work but only on ipad.  iphone is still showing transparent.  I am using the regular picker, not the date picker.

Commit:
Working in iOS8 (64-bits)

- iOS8 (64-bits)
- Datepicker on iPhone (Fixed)
- Dialog list on iPhone (Fixed)

esdebon committed:
Update README.md

esdebon committed:
iOS 8 Fixed

- Rebuild Binary
- iOS 8 (64-bits)
- Datepicker on iPhone (Fixed)
- Dialog list on iPhone (Fixed)

Commit:
iOS Landscape Toast, Datepicker and List Fixed

iOS Landscape Toast, Datepicker and List Fixed

esdebon committed:
Toast orientation fix iOS 6 and iOS9

Toast orientation fix iOS 6 and iOS9

esdebon committed:
iOS Darkmode

Detection of Dark Mode in iOS

esdebon committed:
Updated

Working in AIR 33 Android and iOS

esdebon committed:
Update README.md

Import data
